### DISCUSSION TITLE
Moving to OVOS from mycroft on stand alone x86

### DISCUSSION THREAD

**Discussion by ether8unny** [OpenVoiceOS/ovos-buildroot#139] [Q&A] [answered]
Is it advisable to wait to make the move from mycroft on an x86 to OVOS? I have mycroft currently on multiple systems and integrated with home assistant using mycroft lounge to handle multiple instances of mycroft. I see the x86 version of OVOS is still a work in progress. I am about to start preparing code for new skills and dont want to repeat the process again for OVOS if can be avoided, but also want the setup im using to be stable. Any advice?

**Comment by JarbasAl** [comment]
x86 support in this repo means a buildroot image you can boot in a x86 system,

if you are just installing ovos software and not using a pre built image then there is no issue running in x86, in fact most of us develop in x86 and there are known x86 setups in the wild (eg, https://www.youtube.com/watch?v=cepLKDtvkPI )

re skills, any skill you develop for "classic core" should work in ovos-core out of the box, if it doesn't please flag that with us as it will be a bug

we have a new CommonIOT integration roadmaped for ovos-core 0.0.9 that will mostly deprecate the existing HA implementations, but that will take some time to get going, the mycroftAI skill should work fine meanwhile and there is also a PHAL plugin for HA providing a nice UI on screen  (no voice yet)

**Reply by ChanceNCounter** [reply]
To be clear: skills you developed for classic, pre-Dinkum Mycroft-core are expected to run on our Assistant.

The smart speaker image from MycroftAI, itself called Classic Core, is unrelated.

**Reply by ether8unny** [reply]
also to make sure i have clarity, the PHAL will interact directly with Home Assistant via a remote(?) gui, i.e. the OVOS gui but the voice aspect of it is not yet functional?

**Comment by j1nx** [comment]
A X86_64 buildroot based image is already being worked on, however stalled a bit because of resources availability. However in the (near) future there will be an X86_64 based firmware image for both physical machines as well as Open Virtual appliances to be loaded by VMWare, VirtualBOX, QEMU, etc.

Do not have a timing for it as of yet. Our new build server would greatly help in speeding things up and freeing up resources.

**Comment by ChanceNCounter** [comment]
Now that we have consolidated discussions beneath our master GitHub org, it's no longer obvious why answers to the original question talked about Buildroot.

The broadest interpretation of this question - should I wait to migrate my Linux-based, `git clone`d instance of Mycroft-core to OVOS? - is difficult to answer. The smoothness of that transition will depend on your distro and your system-level Python version.

In general, if you consider yourself a confident and competent Linux user, there should be no problem. There probably won't be much trouble anyway, but I am much, *much* less confident in how a typical end user would feel just at the moment.

**Comment by builderjer** [comment]
If you have a running system, you can try [this script](https://github.com/OpenVoiceOS/raspbian-ovos/blob/main/manual_install.sh). It should install the basic system without gui.  You can then install the gui from there.  But, this should get the core system installed.  If it doesn't, please [create an issue](https://github.com/OpenVoiceOS/raspbian-ovos/issues) so I can fix it.

**Comment by JarbasAl** [comment, marked as answer]
The easiest way to install OVOS on top of an existing OS is https://github.com/OpenVoiceOS/ovos-installer

In the future buildroot images will exist for standalone x86, some early proof of concepts https://www.youtube.com/shorts/xSoDF_HKpzo
